File: test_julia/random62/Matlab/operand_generator.m
function [out] = operand_generator()
    import MatrixGenerator.*;
    out{ 1 } = generate([1650,1650], Shape.Symmetric(), Properties.SPD());
    out{ 2 } = generate([1650,1700], Shape.General(), Properties.Random([-1, 1]));
    out{ 3 } = generate([1700,1650], Shape.General(), Properties.Random([-1, 1]));
    out{ 4 } = generate([1650,1650], Shape.UpperTriangular(), Properties.Random([10, 11]));
end

File: test_julia/random62/Julia/operand_generator.jl
using MatrixGenerator

function operand_generator()
    M345::Symmetric{Float64,Array{Float64,2}} = generate((1650,1650), [Shape.Symmetric, Properties.SPD])
    M343::Array{Float64,2} = generate((1650,1700), [Shape.General, Properties.Random(-1, 1)])
    M344::Array{Float64,2} = generate((1700,1650), [Shape.General, Properties.Random(-1, 1)])
    M342::UpperTriangular{Float64,Array{Float64,2}} = generate((1650,1650), [Shape.UpperTriangular, Properties.Random(10, 11)])
    return (M345, M343, M344, M342,)
end